### ISSUE TITLE
Content request: Component lifecycle

### ISSUE THREAD

**Issue by notlee** [Financial-Times/origami-website#377]
We lost component lifecycle documentation with the removal of the specification.

https://origami.ft.com/spec/v1/components/#component-lifecycle

It should be reviewed and moved to the docs section, and include missing information such as how to release a component, backport a feature, and any other related topics.

**Comment by chee** [comment]
yes, let's have a meeting about this and see what we think this should be.

let's revisit the idea of replacing our `supportStatus` concept with biz-ops project status

let's also document our use of semver, and in contrast to the old doc let's also explain _why_ a major bump to an existing dependency requires a major bump to the dependent.
